atlasdb-processors/src/main/java/com/palantir/processors/AutoDelegateProcessor.java

```diff
@@ -96,7 +96,7 @@ public Set<String> getSupportedAnnotationTypes() {
 
     @Override
     public SourceVersion getSupportedSourceVersion() {
-        return SourceVersion.RELEASE_11;
+        return SourceVersion.RELEASE_17;
```

Comment:
we probably don't want to bump this every time we upgrade the source compat target
```suggestion
        return SourceVersion.latestSupported();
```